tests/features/workspace.feature

```diff
@@ -0,0 +1,147 @@
+
```

Comment:
We should add a test where Workspace is instantiated with token 123 and there is no configuration available for token 123 (in this case we will get error when calling `start` or `config`).

Add also a test where there is a configuration for token 123 but it's format is not valid, in this case also we will get error when calling `start` or `config`.